resources/lib/token_decoder.py

```diff
@@ -0,0 +1,79 @@
+import re
+import math
+
+
+# the + in the beginning parses text to a number

```

Comment:
neat :)
i understood how it works but just threw it inside javascript VM and got the cookie inside document.
is it still decodes to `document.cookie=....` as it was few weeks ago?